Sources/Kitura/RouterResponse.swift

```diff
@@ -238,9 +241,12 @@ public class RouterResponse {
     /// is the alphanumeric characters and `[]$._`).
     /// - Returns: this RouterResponse.
     public func send(jsonp: JSON, callbackParameter: String = "callback") throws -> RouterResponse {
+        if RouterResponse.sanitizeJSIdentifierRegex == nil {
```

Comment:
@shmuelk I may be wrong, but I think `static private let oneTime = OneTimeInitializations()` in KituraNet.ClientRequest is actually currently broken. 

Swift lazily initializes class variables and constants and since `oneTime` is never referenced anywhere, it never actually gets initialized and so `curl_global_init(Int(CURL_GLOBAL_SSL))` in `OneTimeInitializations.init()` never gets invoked.